docs/content/kv_commands.md

```diff
@@ -341,11 +341,21 @@ yes
 
```

Comment:
these docs changes can go, we're removing bucket commands

Reply:
If the readme for KV still exists does it hurt to improve it though? Not a big change happy to remove if you think so

Reply:
all the improvements you added are to commands that are going to be deleted, though.